src/Framework/Interaction/ScatteringType.h

```diff
@@ -78,6 +80,8 @@ class ScatteringType
       case(kScInverseBetaDecay) :        return "IBD";       break;
       case(kScGlashowResonance) :        return "GLR";       break;
       case(kScIMDAnnihilation) :         return "IMDAnh";    break;
+      case(kScPhotonRES) :               return "PhotonRES"; break;
```

Comment:
Perhaps, can we have some better naming uniformity here? We have 'kScGlashowResonance', and 'kScPhotonRES'. We have 'kScCoherentProduction' and 'kScPhotonCOH'. Maybe we can fully spell out the 'resonance', 'coherent' in the kSc** enum, and use the short 'RES', 'COH' version in the string that this code returns?

Reply:
Sorry for the delay. I will make the changes in the naming convention.